operations/enterprises.md

```diff
@@ -21,7 +36,17 @@ Returns all company profiles of the enterprise, possible filtered by their ident
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `Ids` | array of string | optional | If specified, returns only companies with the specified identifiers. |
+| `Ids` | array of string | optional | Unique identifiers of [Company](enterprises.md#company)s. |
+| `Names` | array of string | optional | Names of [Company](enterprises.md#company)s. |
+| `CreatedUtc` | [Time Filter](enterprises.md#time-filter) | optional | Interval of [Company](enterprises.md#company)s creation date and time. |
+| `UpdatedUtc` | [Time Filter](enterprises.md#time-filter) | optional | Interval of [Company](enterprises.md#company)s last update date and time. |
```

Comment:
Shouldn't this DTO be called Interval, time interval or date time interval? Time filter sounds too general. Also if you look at JSON, there are duplicate suffixes.

Reply:
@siroky what do you mean by those suffixes?

Reply:
@siroky  Do you mean Utc in the property name and then UTC in the start and the end? That's how we designed and it's used in others endpoints.